pkg/tests/upgrade_tests/upgrade_test.go

```diff
@@ -40,8 +40,10 @@ import (
 )
 
 var (
-	testDatabase       = flag.String("database", "tmp_db_timescale_upgrade_test", "database to run integration tests on")
-	printLogs          = flag.Bool("print-logs", false, "print TimescaleDB logs")
+	testDatabase = flag.String("database", "tmp_db_timescale_upgrade_test", "database to run integration tests on")
+	printLogs    = flag.Bool("print-logs", false, "print TimescaleDB logs")
+	// use "local/dev_promscale_extension:head-ts2-pg13" for local testing
+	dockerImage        = flag.String("image", "ghcr.io/timescale/dev_promscale_extension:develop-ts2-pg14", "docker image for database")
```

Comment:
We may need to pass this in the cicd GitHub Actions code. Let's ask @JamesGuthrie what the plan here is.